receiver/kafkareceiver/kafka_receiver_test.go

```diff
@@ -306,7 +306,7 @@ var _ sarama.ConsumerGroup = (*testConsumerGroup)(nil)
 
 func (t testConsumerGroup) Consume(ctx context.Context, topics []string, handler sarama.ConsumerGroupHandler) error {
 	t.once.Do(func() {
-		handler.Setup(testConsumerGroupSession{})
+		t.err = handler.Setup(testConsumerGroupSession{})
 	})
 	return t.err
 }
```

Comment:
I think this is the way it was intended to be written but I am not 100% sure, please check